.coveragerc

```diff
@@ -1,10 +1,29 @@
+# Generated from:
+# https://github.com/gocept/meta/tree/master/config/pytest
 [run]
-branch = True
 source = gocept.reference
+plugins = coverage_python_version
+branch = true
+parallel = true
```

Comment:
This should not be necessary. IMHO it is only needed when running multiple processes during a test run.